### DISCUSSION TITLE
Cycling TextBlock

### DISCUSSION THREAD

**Discussion by Arlodotexe** [CommunityToolkit/Labs-Windows#543] [Ideas]
From the Voice Recorder app on Windows 11:

https://github.com/CommunityToolkit/Labs-Windows/assets/9384894/80033067-c5ae-4724-9a23-ca121e1690b6

A rather simple control that could be useful in a number of scenarios but would be nontrivial to implement each time. This seems like a good candidate for a reusable Toolkit Labs experiment.

**Comment by Arlodotexe** [comment]
The progress ring should be optional or separate.

**Comment by Arlodotexe** [comment]
@Avid29 maybe this could fit into the marquee control you made?

**Reply by 0x5bfa** [reply]
Microsoft Store also has this feature vertically in Install button when installing.
Vertical version would be useful as well.

**Reply by Avid29** [reply]
> Microsoft Store also has this feature vertically in Install button when installing. Vertical version would be useful as well.

The `MarqueeText` control already features a MarqueeDirection field which could be used to make it go vertical instead of horizontal.
